examples/run_nutation.py

```diff
@@ -82,16 +82,17 @@
         SW_kHz=config_dict["SW_kHz"],
         ret_data=data,
     )
-data.setaxis("indirect", p90_range_us * 1e-6).set_units("indirect", "s")
+data.rename("indirect", "p_90")
+data.setaxis("p_90", p90_range_us * 1e-6).set_units("p_90", "s")
 # {{{ chunk and save data
-data.chunk("t", ["ph1", "t2"], [4, -1])
-data.setaxis("ph1", ph1_cyc / 4)
+data.chunk("t", ["ph2", "ph1", "t2"], [2, 2, -1])
+data.setaxis("ph1", ph1_cyc / 4).setaxis("ph2", ph2_cyc / 4)
 if config_dict["nScans"] > 1:
     data.setaxis("nScans", r_[0 : config_dict["nScans"]])
-data.reorder(["ph1", "nScans", "t2"])
+data.reorder(["nScans", "ph2", "ph1", "p_90", "t2"])
 data.set_units("t2", "s")
-data.set_prop("postproc_type", "spincore_nutation_v4")
-data.set_prop("coherence_pathway", {"ph1": +1})
+data.set_prop("postproc_type", "spincore_nutation_v3")
```

Comment:
I looked through the load_data and there is already a postprocessing function that suits this phase cycling!

Reply:
I would call it 5, I think.

Part of the function of this is to label the version number of the ppg, and we're not sure that this does the same thing as v3

In the more recent versions of procscripts, I have the generic post-processing function which should handle most cases....  So we would just end up adding a line to the dictionary